### PR TITLE
Update creating geopackage

### PR DESCRIPTION
**Pull request by jansule**
This reads the custom sql for layers from files.

Also added enum for job status.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +49,13 @@ that allows following additional configs:
 - `ORKA_DB_MIN_CONNECTION` = application database min connections
 - `ORKA_DB_MAX_CONNECTION` = application database max connections
 - `ORKA_GPKG_PATH` = path to where the created gpkg files should be placed
-- `ORKA_LAYERS_YML` = path to yaml file that contains the layer specs. This file must be located within the instance folder 
+- `ORKA_LAYERS_PATH` = path to folder containing the layer sqls. This file must be located within the instance folder
```
Comment (simonseyock) [inline README.md]:
```suggestion
- `ORKA_LAYERS_PATH` = path to folder containing the layer sqls. This directory must be located within the instance folder 
```

File: README.md [hunk 2]
```diff
@@ -41,9 +49,13 @@ that allows following additional configs:
 - `ORKA_DB_MIN_CONNECTION` = application database min connections
 - `ORKA_DB_MAX_CONNECTION` = application database max connections
 - `ORKA_GPKG_PATH` = path to where the created gpkg files should be placed
-- `ORKA_LAYERS_YML` = path to yaml file that contains the layer specs. This file must be located within the instance folder 
+- `ORKA_LAYERS_PATH` = path to folder containing the layer sqls. This file must be located within the instance folder 
 - `ORKA_THREAD_TIMEOUT` = timeout in seconds after which a running thread should be killed.
 - `ORKA_MAX_THREADS` = number of allowed threads
+- `ORKA_LOG_FILE` = path to log file
+- `ORKA_STYLE_PATH` = path to the file that contains all styles, etc.
+- `ORKA_STYLE_FILE` = name of the zip file (including `.zip`) that contains all styles, etc.
```
Comment (simonseyock) [inline README.md:57]: Do we need both paths?

File: orka_vector_api/helper/gdal_helper.py [hunk 3]
```diff
@@ -18,66 +19,57 @@ def _get_geopackage_cmd(filename, layername, sql, host=None, port=None, database
...
+    file_name = os.path.abspath(os.path.join(gpkg_path, data_id + '.gpkg'))
+    layer_sqls = _get_layer_sqls(layers_path)
+    for layer_name, layer_sql in layer_sqls.items():
+        if timeout_e is not None and timeout_e.isSet():
             break
-        cmd = _get_geopackage_cmd(filename, layername, layer_sql, **db_props)
-        # app.logger.info(cmd)
+        gpkg_sql = _get_gpkg_sql(layer_sql, bbox)
+        gpkg_sql_escaped = _escape_sql(gpkg_sql)
+        print(gpkg_sql_escaped)
```
Comment (simonseyock) [inline orka_vector_api/helper/gdal_helper.py:30]: should this be a logging statement?

File: orka_vector_api/helper/job_helper.py [hunk 4]
```diff
@@ -150,6 +157,25 @@ def threads_available(conn, app):
...
 
 
+def bbox_size_allowed(conn, app, bbox):
+    minx = float(bbox[0])
+    miny = float(bbox[1])
+    maxx = float(bbox[2])
+    maxy = float(bbox[3])
+
+    max_area = app.config['ORKA_MAX_BBOX']
+    q = 'SELECT ST_AREA(st_transform(l.geom, 3857))/1000000 from (select \'SRID=4326;POLYGON((%(minx)s %(miny)s,%(maxx)s %(miny)s, %(maxx)s %(maxy)s,%(minx)s %(maxy)s,%(minx)s %(miny)s))\' :: geometry geom) as l;'
```
Comment (simonseyock) [inline orka_vector_api/helper/job_helper.py]: can't you use `ST_MakeEnvelope` here? it also accepts a srid parameter http://postgis.net/docs/ST_MakeEnvelope.html

**Review by simonseyock — APPROVED** [review]
Very nice! Just some minor remarks

File: orka_vector_api/helper/job_helper.py (hunk 4 above)
Comment (jansule) [reply orka_vector_api/helper/job_helper.py]: Good point. This should be the better approach. will change this accordingly

File: orka_vector_api/helper/gdal_helper.py (hunk 3 above)
Comment (jansule) [reply orka_vector_api/helper/gdal_helper.py:30]: As we run this in a thread, we do not have access to the application. So we have to add a different logging mechanism here

**Pull request merged**